build.sbt

```diff
@@ -19,8 +19,9 @@ import uk.gov.hmrc.DefaultBuildSettings
 
 val appName = "individuals-state-benefits-api"
 
-ThisBuild / scalaVersion := "2.13.16"
+ThisBuild / scalaVersion := "3.5.2"
 ThisBuild / majorVersion := 0
+ThisBuild / scalacOptions ++= Seq("-Werror", "-Wconf:msg=Flag.*repeatedly:s")
```

Comment:
having "-Werror" here means you do not need the 2 instances of "-Xfatal-warnings" below, can we get them removed please?